src/GalleryViewer/GalleryViewer.tsx

```diff
@@ -25,7 +25,7 @@ const onInit = async () => {
 
   const props: GalleryAndNotebookViewerComponentProps = {
     junoClient: new JunoClient(),
-    selectedTab: galleryViewerProps.selectedTab || GalleryTab.PublicGallery,
+    selectedTab: galleryViewerProps.selectedTab || GalleryTab.OfficialSamples,
```

Comment:
Is this change needed cos public gallery is hidden currently? since public gallery is behind a flight, should we check for the same flight here to choose between public gallery/official samples?

Reply:
Good point. I'll add a feature flag check